Cargo.toml

```diff
@@ -99,15 +97,15 @@ all = ["multi-user", "rta", "static-openssl"]
 hsm-tests-kmip = ["hsm"]
 hsm-tests-pkcs11 = ["hsm"]
 
+[dev-dependencies]
+regex    = "1.5.5"
```

Comment:
Why put regex in dev-dependencies when it is already in the main dependencies with the same version number?

Reply:
Ah it was already there, you just moved it...

Reply:
And ah it is optional, okay I've answered my own question :-)